evap/staff/views.py

```diff
@@ -592,11 +592,22 @@ def course_edit(request, semester_id, course_id):
 @staff_required
 def helper_course_edit(request, semester, course):
     @receiver(RewardPointGranting.granted_by_removal)
-    def notify_reward_points(users, **kwargs):
-        names = ", ".join('"{}"'.format(user.username) for user in users)
-        messages.info(request, ungettext("The removal of participants has granted {affected} user reward points: {names}.",
-            "The removal of participants has granted {affected} users reward points: {names}.",
-            len(users)).format(affected=len(users), names=names))
+    def notify_reward_points(granted, **kwargs):
+        transactions = ', '.join(
+            ngettext(
+                '{transaction.points} point to "{transaction.user.username}"',
+                '{transaction.points} points to "{transaction.user.username}"',
+                transaction.points
+            ).format(transaction=transaction)
+            for transaction in granted
+        )
+        messages.info(request,
+            ngettext(
+                "The removal of participants has granted {affected} user reward points: {transactions}.",
+                "The removal of participants has granted {affected} users reward points: {transactions}.",
+                len(granted)
+            ).format(affected=len(granted), transactions=transactions)
+        )
```

Comment:
we could keep this really simple and just emit one message per user. i suppose having multiple users here happens rarely and the simple version is good enough, therefore i would prefer that. @janno42?

Reply:
one message per user is fine.